app/src/test/java/casino/bettingsystem/BettingSystemTest.java

```diff
@@ -0,0 +1,71 @@
+package casino.bettingsystem;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import casino.shared.BettingSystem;
+
+public class BettingSystemTest {
+
+    @Test
+    public void testPositiveDeposit() {
+        BettingSystem bet = new BettingSystem(100);
+        bet.deposit(10);
+        bet.deposit(20);
+        bet.deposit(30);
+
+        assertEquals(160, bet.getBalance());
+    }
+
+    @Test
+    public void testNegativeDeposit() {
+        BettingSystem bet = new BettingSystem(100);
+        bet.deposit(-50);
+
+        assertEquals(100, bet.getBalance());
```

Comment:
Assuming negative deposits are not withdrawals and should be ignored? Should it throw an exception instead of doing nothing?